### ISSUE TITLE
[Core feature]  Execution Cluster label creation

### ISSUE THREAD

**Issue by gdabisias** [flyteorg/flyte#4029]
### Motivation: Why do you think this is important?

It is tedious to create Execution Cluster Labels using attrFiles.
Current process is outlined [here](https://docs.flyte.org/en/latest/deployment/deployment/multicluster.html#deployment-deployment-multicluster) and when multiple projects and environments need to be labeled, then this is not an ideal process

### Goal: What should the final outcome look like, ideally?

Ideally this should be either done with one file with all mappings or adding the option to pass the attrFile attributes directly as parameters to flytectl instead of passing a file

### Describe alternatives you've considered 

I created the files manually

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by fg91** [comment]
@wild-endeavor I remember that somewhere in slack I read that one can control the execution cluster label for each project via the helm values but I can't find the conversation anymore, also in the archive.

Do you know whether this works or whether I'm mistaken here?

Ideally I would want to control which dataplane clusters are used for which project purely with infrastructure as code as opposed to imperative flytectl commands.

**Comment by gdabisias** [comment]
@fg91 I agree that ideally this should be also possibly through infrastructure provisioning. In general all settings should be editable from there to ensure that users can use CI/DB pipelines to deploy changes, without needing to directly interact with flytectl
